Automattic/wp-calypso
# Editing Toolkit: Load patterns from the rest API endpoint v2

**This is a modified version of #45926 that loads the block patterns only when the user is loading the block editor.**

-----

Remove block patterns from within the codebase, and load them via the new rest API endpoint.

#### Testing instructions

* Load the editor and the patterns inserter and confirm you see all the following categories:

<img width="351" alt="Screen Shot 2020-10-08 at 12 50 05 PM" src="https://user-images.githubusercontent.com/1464705/95506673-11ce1f00-0965-11eb-819b-ff6267d87e41.png">

* Try inserting a pattern and confirm this works as expected.
* Switch your blog to a different language (settings->general)
* Confirm that you can still see patterns in the pattern inserter
* Some patterns may be translated (still confirming this).
